site/docs/v1/tech/identity-providers/samlv2-idp-initiated/index.adoc

```diff
@@ -0,0 +1,118 @@
+
+If you must enable this feature to support a legacy integration or because the user experience is of the highest concern, then proceed.
+====
+
+Typically, a SAML flow begins at a service provider such as FusionAuth. In a service provider initiated login, the SAML Authentication Response is returned to the service provider in response to an authentication request. FusionAuth can validate certain parameters and only accept a response for a request that FusionAuth initiated. 
+
+In contrast, during an identity provider initiated login, the third-party identity provider sends FusionAuth a SAML Authentication Response indicating a user has been authenticated. FusionAuth did not initiate this request so it is unexpected and FusionAuth does not have the full context. This limits what can be verified and requires FusionAuth to allow a cross site request from this third party. 
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+* <<Create a SAML v2 IdP Initiated Identity Provider>>
+* <<Integration Details>>
+* <<CORS Configuration>>
+
+We also provide specific examples for configuring SAML with some providers whose implementation requires unique configuration. If you'd like us to provide additional examples, please open a request on https://github.com/FusionAuth/fusionauth-site/issues[GitHub].
+
+* link:/docs/v1/tech/identity-providers/samlv2/okta/[Configure SAML v2 IdP Initiated SSO for Okta]
+
+Once you have completed this configuration you will be able to log in to FusionAuth directly from a SAMLv2 IdP. 
+
+
+== Create a SAML v2 IdP Initiated Identity Provider
+
+To create an Identity Provider navigate to [breadcrumb]#Settings -> Identity Providers# and click [breadcrumb]#Add provider# and select [breadcrumb]#SAML v2 IdP Initiated#.
+
+This will take you to the [breadcrumb]#Add SAML v2 IdP Initiated# panel. Here you will need to fill out the required fields. 
+
+image::identity-providers/identity-provider-samlv2-idp-initiated-add.png[Add SAML v2 IdP Initiated,width=1200,role=bottom-cropped]
+
+
+=== Form Fields
+
+[.api]
+[field]#Enabled# [Required]#Required#::
+Determines if this provider is enabled. If it is false then it will be disabled globally.
+
+[field]#Id# [optional]#Optional#::
+An optional UUID. When this value is omitted a unique Id will be generated automatically.
+
+[field]#Name# [required]#Required#::
+A unique name to identify the identity provider. This name is for display purposes only and it can be modified later if desired.
+
+[field]#Issuer# [required]#Required#::
+The EntityId (unique identifier) of the SAML v2 identity provider. This value should be provided to you.
+
+[field]#Use NameId for email# [optional]#Optional#::
+If this is enabled, FusionAuth will assume that the `NameID` in the SAML response contains the email address of the user.
```

Comment:
consider
If this is enabled, FusionAuth will _**expect**_ the `NameID` in the SAML response contains the email address of the user.